cases/management/commands/enrich_ciaa_timeline.py

```
@@ -0,0 +1,421 @@
+            self.stdout.write(
+                self.style.WARNING("  No special:* court case reference found — skipping")
+            )
+            return
+
+        court_identifier, case_number = special_ref.split(":", 1)
+
+        logger.debug("  Fetching NGM data for %s:%s", court_identifier, case_number)
+
+        try:
+            case_details = get_court_case_details(court_identifier, case_number)
+        except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)